specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/InitiateTransfer.json

```diff
@@ -1,6 +1,6 @@
 {
   "parameters": {
-    "api-version": "2018-08-01-preview",
+    "api-version": "2018-11-01-preview",
     "billingAccountName": "billingAccountName",
     "invoiceSectionName": "invoiceSectionName",
```

Comment:
Below: recipient/initiator email ids should be different.
invoiceSectionId to invoiceSectionName

Reply:
Done